### PR TITLE
Central working case study

### PR DESCRIPTION
**Pull request by omonk**
## Description - [Trello ticket](https://trello.com/c/2W01i4Ib/539-case-study-central-working-groomed)

WIP: 

- Need to take BlockRow and includes updates from the Doctorlink PR

### CODE REVIEW

File: src/components/Common/BrownBackground.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components'
+
+export default styled.section`
+  background-color: ${({ theme }) => theme.colors.black};
+`
```
Comment (GBouffard) [inline src/components/Common/BrownBackground.js]: Not a change or anything but `BrownBackground.js` and `theme.colors.black`??
Our legacy naming conventions in theme are so misleading. I would actually suggest a ticket to rename them.

File: src/pages/case-study/central-working-changing-how-you-work.js [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+    ...props
+  }))
+
+const normalise = (arr = [], index = 0) => normaliseAll(arr, index)[index]
+
+const GreyBodyPrimary = styled(BodyPrimary)`
+  color: ${({ theme }) => theme.colors.secondaryText};
+`
+
+const Block3Col = styled(Col)`
```
Comment (GBouffard) [inline src/pages/case-study/central-working-changing-how-you-work.js]: Block3Col, 4 and 6 are exactly the same. I would rename to something like BlockColPaddingTop.

File: src/pages/case-study/central-working-changing-how-you-work.js [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+      <GreyBackground>
+        <Grid>
+          {/* Outcomes */}
+          <BlockRow
+            style={{ flexDirection: 'column', alignItems: 'center' }}
+            mobile={{ bottom: '4', top: '4' }}
+            tablet={{ top: '6' }}
+          >
+            {shouldRender(data2) && (
+              <Fragment>
```
Comment (GBouffard) [inline src/pages/case-study/central-working-changing-how-you-work.js]: You don't need a Fragment here.

File: src/pages/case-study/central-working-changing-how-you-work.js [hunk 4]
```diff
@@ -0,0 +1,428 @@
...
+          </BlockRow>
+          <BlockRow
+            mobile={{ bottom: '5' }}
+            smallTablet={{ bottom: '4' }}
+            tablet={{ bottom: '6' }}
+          >
+            {outComesDataFigures.map(({ text }, index) => (
+              <Col
+                width={[1, 1, 1, 1, 4 / 12]}
+                style={{
```
Comment (GBouffard) [inline src/pages/case-study/central-working-changing-how-you-work.js]: Why use inline-style here? I think it should be some kind of StyledCol to keep it consistent and use styled-component

File: src/pages/case-study/central-working-changing-how-you-work.js [hunk 5]
```diff
@@ -0,0 +1,428 @@
...
+                renderers={{
+                  // eslint-disable-next-line
+                  heading: props => <SectionTitle reverse {...props} />
+                }}
+                source={normalise(data8).text}
+              />
+            </Col>
+          </BlockRow>
+          <BlockRow style={{ justifyContent: 'center' }}>
+            <Col width={[1, 1, 1, 1, 10 / 12]} style={{ alignSelf: 'center' }}>
```
Comment (GBouffard) [inline src/pages/case-study/central-working-changing-how-you-work.js]: Same comment as above for inline-style

File: src/pages/case-study/central-working-changing-how-you-work.js (hunk 4 above)
Comment (omonk) [reply src/pages/case-study/central-working-changing-how-you-work.js]: errr yeah I didn't want to create a specific styled component for this. Unfortunately even though Col is a `<Flex>` component the flex api hasn't been implemented which means we either  have to write inline or create a styled component

File: src/pages/case-study/central-working-changing-how-you-work.js (hunk 2 above)
Comment (omonk) [reply src/pages/case-study/central-working-changing-how-you-work.js]: Only 4 and 6 are the same no? 

I would have created a styled component here but not sure what to call it. `BlockColPaddingTop` is super vague in this context.

**Pull request closed without merging**